12-solid/3-liskov-substitution/0-notes.md

```diff
@@ -0,0 +1,32 @@
+# Liskov substitution principle
+
+![Illustration: Liskov substitution principle](../storage/3-intro-liskov-substitution-principle.png)
```

Comment:
I kinda like this image more even I used the duck example

![image](https://user-images.githubusercontent.com/12962418/66259040-3b8cbd00-e7ca-11e9-9f2a-aa8992cf78e5.png)

What do you guys think (say yes so my  inner nerd be satisfied :smiley: )

Reply:
For sure

Reply:
we'll change it later if needed, for now I'll keep the existing one, i feel it's bit expressive than this one, let's see